dockstore-webservice/src/main/java/io/dockstore/webservice/core/WorkflowVersion.java

```diff
@@ -106,18 +106,19 @@ public boolean equals(Object obj) {
             return false;
         }
         final WorkflowVersion other = (WorkflowVersion)obj;
-        return Objects.equals(super.getName(), other.getName()) && Objects.equals(super.getReference(), other.getReference());
+        return Objects.equals(this.getName(), other.getName()) && Objects.equals(this.getReference(), other.getReference());
     }
 
     @Override
     public int compareTo(WorkflowVersion that) {
+        // super.compareTo(that);
         return ComparisonChain.start().compare(this.getName(), that.getName(), Ordering.natural().nullsLast())
                 .compare(this.getReference(), that.getReference(), Ordering.natural().nullsLast()).result();
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(name, reference);
+        return Objects.hash(super.hashCode());
```

Comment:
At this point I would just not override the method, instead of hashing the hash.